### PR TITLE
Add support for py310

### PR DESCRIPTION
**Pull request by ssbarnea**
Related: https://github.com/ansible-community/devtools/issues/6

### CODE REVIEW

**Review by webknjaz — CHANGES_REQUESTED** [review]
I've noticed a lot of unrelated formatting changes. Drop them.

File: .github/workflows/tox.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-        - 3.9
+        - "3.9"
```
[the remark this answers is not in the document]
Comment (ssbarnea) [reply .github/workflows/tox.yml]: I will do the formatting in a follow-up.

File: .github/workflows/tox.yml [hunk 2]
```diff
@@ -131,12 +132,18 @@ jobs:
           os: ubuntu-20.04
           python-version: 3.9
           devel: true
+        - tox_env: py310
+          os: ubuntu-20.04
+          python-version: 3.10
```
Comment (tadeboro) [inline .github/workflows/tox.yml:137]:
```suggestion
          python-version: "3.10"
```

**Review by tadeboro — CHANGES_REQUESTED** [review]

**Review by tadeboro — APPROVED** [review]

**Review by webknjaz — APPROVED** [review]

File: .github/workflows/tox.yml [hunk 3]
```diff
@@ -194,6 +199,7 @@ jobs:
       env:
         TOXENV: ${{ matrix.tox_env }}-core
     - name: "Test with tox: ${{ matrix.tox_env }}-ansible29"
+      if: ${{ !matrix.skip_ansible29 }}
```
[the remark this answers is not in the document]
Comment (ssbarnea) [reply .github/workflows/tox.yml:202]: I did not see this in GHA examples. Seems like overkill but I am curious to learn about the subject as I did find conditionals with GHA as having some unexpected behaviors at times.

**Pull request merged**